src/blocks/blocks/form/edit.js

```diff
@@ -39,9 +40,13 @@ import metadata from './block.json';
 import { blockInit } from '../../helpers/block-utility.js';
 import Inspector from './inspector.js';
 import Placeholder from './placeholder.js';
+import { getListIdOptionFrom } from './integrations';
+import { key } from '@wordpress/icons/build-types';
```

Comment:
As a future note: imports added by VS Code when miss clicking in autocomplete can make the script fail to load and no error will be given in Console.